### ISSUE TITLE
Add transpose operator

### ISSUE THREAD

**Issue by pditommaso** [nextflow-io/nextflow#440]
The `transpose` operator implements the opposite function of `groupTuple`. For example given a channel emitting the following tuples: 

```
[ key1, [value1, value2] ] 
[ key2, [value3, value4] ] 
```

apply `transpose` it will return the following: 

```
[ key1, value1 ]
[ key1, value2 ] 
[ key2, value3 ] 
[ key2, value4 ]
```

**Comment by pditommaso** [comment]
More complex examples: 

```
[ a, [p,q], [x,y] ]
[ b, [s,t], [x,y] ] 
```
the `transpose` returns: 
```
[ a, p, x ] 
[ a, q, y ] 
[ b, s, x ] 
[ b, t, y ] 
```

given:
```
[ a, [p,q], [x,y,w,z] ]
[ b, [s,t], [x,y,w,z] ] 
``` 
It returns: 
```
[ a, p, x ]
[ a, q, y ]
[ b, s, x ]
[ b, t, x ]
```

For the same input, specifying `remainder: true` it returns: 

```
[ a, p, x ]
[ a, q, y ]
[ a, null, w ]
[ a, null, z ]

[ b, s, x ]
[ b, t, x ]
[ b, null, w ]
[ b, null, z ]
```

**Comment by pditommaso** [comment]
Included in version `0.26.0-beta3`
